oauth2server/proj/settings/docker_settings.py

```diff
@@ -20,14 +24,40 @@
 # }
 
 
+# Extract the database URI value from VCAP_SERVICES
+
+
+if 'VCAP_SERVICES' in os.environ:
+    remoteLogger.info('VCAP_SERVICES found in os.environ')
+    decoded_config = json.loads(os.environ['VCAP_SERVICES'])
+    for key, value in decoded_config.items():
+        remoteLogger.debug("Inspecting key: {} with value: {}".format(str(key), str(value)))
+        if decoded_config[key][0]['name'] == 'postgresql':
+            creds = decoded_config[key][0]['credentials']
+            uri = creds['uri']
+            remoteLogger.info("Found postgres uri string in vcap settings")
+            remoteLogger.info("Postgres DATABASE uri: {}".format(uri))
+
+
+
+else:
+    remoteLogger('VCAP_SERVICES NOT found in os.environ using default SQL database')
```

Comment:
Not completely sure how the logging levels work here sorry, but does this need an explicit level when setting the logging message? E.g. 'info' level?